assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx

```diff
@@ -625,8 +625,8 @@ const MultipleChoiceQuestion: React.FC<PartComponentProps<McqModel>> = (props) =
     if (shouldSave) {
       saveState({
         numberOfSelectedChoices: newCount,
-        selectedChoice: newSelectedChoices.slice(-1)[0] || 0,
-        selectedChoiceText: updatedChoicesText.slice(-1)[0] || '',
+        selectedChoice: newSelectedChoices.length ? newSelectedChoices.sort()[0] : 0,
```

Comment:
As per SS, It does not matter in which order you select the MCQ option (Multi select), the lowest one in the array gets set to 'selectedChoice' property.